### PR TITLE
extract wa training data & bootstrap STT test set

### PR DESCRIPTION
**Pull request by andrewrfreed**
See #9 issue for details.

Two parts:
1) Extract WA intent/entity data
2) During synthesis, create STT reference file

DCO 1.1 Signed-off-by: Andrew R. Freed [afreed@us.ibm.com](mailto:afreed@us.ibm.com)

### CODE REVIEW

**Review by lmazzoli — APPROVED** [review]
Code changes look good.  Nice addition.

**Review by tnixa — APPROVED** [review]
